### PR TITLE
fix for header when there is no currency assigned

### PR DESCRIPTION
**Pull request by rupato-deriv**

### CODE REVIEW

File: src/botPage/view/deriv/layout/Header/components/tab-content.jsx [hunk 1]
```diff
@@ -70,27 +77,26 @@ const TabContent = ({ tab, isActive, setIsAccDropdownOpen }) => {
...
+                      src={`image/deriv/currency/ic-currency-${demo_account
                         ? "virtual"
-                        : account.currency?.toLowerCase()
-                        }.svg`}
+                        : currency?.toLowerCase()
+                        || "unknown"}.svg`}
                     />
                     <span>
-                      {accounts[acc].demo_account
+                      {(!currency) ? <span className="symbols">No currency assigned</span> : <></>}
```
Comment (maryiafrantsava-binary) [inline src/botPage/view/deriv/layout/Header/components/tab-content.jsx:86]:
```suggestion
                      {!currency && <span className="symbols">No currency assigned</span>}
```

File: src/botPage/view/deriv/layout/Header/components/tab-content.jsx [hunk 2]
```diff
@@ -70,27 +77,26 @@ const TabContent = ({ tab, isActive, setIsAccDropdownOpen }) => {
...
                     </span>
-                    <span className="account__switcher-balance">
-                      {account?.balance?.toLocaleString(undefined, {
-                        minimumFractionDigits:
-                          config.currency_name_map[account.currency]
-                            ?.fractional_digits ?? 2,
-                      })}
-                      <span className="symbols">
+                    <span className="account__switcher-balance test">
+                      {console.log('function called', getBalance())}
```
Comment (maryiafrantsava-binary) [inline src/botPage/view/deriv/layout/Header/components/tab-content.jsx:97]: Are you sure we need this console.log()?

File: src/botPage/view/deriv/layout/Header/components/tab-content.jsx [hunk 3]
```diff
@@ -70,27 +77,26 @@ const TabContent = ({ tab, isActive, setIsAccDropdownOpen }) => {
...
+                      {account.currency ? getBalance() : ""}
```
Comment (maryiafrantsava-binary) [inline src/botPage/view/deriv/layout/Header/components/tab-content.jsx:98]:
```suggestion
                      {account.currency && getBalance()}
```

File: src/botPage/view/deriv/layout/Header/components/account-actions.jsx [hunk 4]
```diff
@@ -74,11 +77,14 @@ const AccountActions = () => {
                 <img
                     id="header__acc-icon"
                     className="header__acc-icon"
-                    src={`image/deriv/currency/ic-currency-${is_virtual ? "virtual" : currency.toLowerCase()}.svg`}
+                    src={`image/deriv/currency/ic-currency-${is_virtual ? "virtual" : currency.toLowerCase() || "unknown"}.svg`}
```
Comment (prince-deriv) [inline src/botPage/view/deriv/layout/Header/components/account-actions.jsx:80]: This is quite long to pass directly as a src value, can we assign it to a variable instead?

File: src/botPage/view/deriv/layout/Header/components/account-dropdown.jsx [hunk 5]
```diff
@@ -92,7 +101,8 @@ const AccountDropdown = React.forwardRef((props, dropdownRef) => {
           <div className="account__switcher-total-balance">
             <span className="account__switcher-total-balance-text">{translate("Total assets")}</span>
             <span className="account__switcher-total-balance-amount account__switcher-balance">
-              {activeTab === "demo" ? getTotalDemo(accounts) : total_deriv.amount}
+              
+              {activeTab === "demo" ? getTotalDemo(accounts) : getBalance}
               <span className="symbols">&nbsp;{activeTab === "demo" ? "USD" : total_deriv.currency}</span>
```
Comment (sara-fs) [inline src/botPage/view/deriv/layout/Header/components/account-dropdown.jsx:106]: Do we need to keep total_deriv here? while we are getting the currency above

File: src/botPage/view/deriv/layout/Header/components/tab-content.jsx (hunk 1 above)
Comment (sara-fs) [inline src/botPage/view/deriv/layout/Header/components/tab-content.jsx:86]: No need translate?

File: src/botPage/view/deriv/layout/Header/components/tab-content.jsx (hunk 2 above)
Comment (sara-fs) [inline src/botPage/view/deriv/layout/Header/components/tab-content.jsx:97]: can we remove console?

Comment (rupato-deriv) [reply src/botPage/view/deriv/layout/Header/components/tab-content.jsx]: removed

File: src/botPage/view/deriv/layout/Header/components/account-dropdown.jsx (hunk 5 above)
Comment (rupato-deriv) [reply src/botPage/view/deriv/layout/Header/components/account-dropdown.jsx]: removed total deriv

**Review by sandeep-deriv — APPROVED** [review]
LGTM 👌

**Review by sara-fs — APPROVED** [review]

**Pull request merged**